app/api/api_v1/schemas/search.py

```diff
@@ -121,50 +125,6 @@ def __get_pydantic_json_schema__(
         json_schema["properties"].pop("all_results")
         json_schema["properties"].pop("documents_only")
 
-        # Extra field documentation
-        # TODO: Move these to parent class
```

Comment:
I can't see where these have been moved to in this PR or are we just deleting?

Reply:
The documentation now comes from the sdk model, the model config allows us to use the docstrings: https://github.com/climatepolicyradar/cpr-sdk/blob/main/src/cpr_sdk/models/search.py#L72